dbt/include/exasol/macros/adapters.sql

```diff
@@ -21,9 +21,9 @@ ALTER_COLUMN_TYPE_MACRO_NAME = 'alter_column_type'
       lower(table_schema) as [schema],
   	  lower(table_type) as table_type
     from (
-		select table_name,table_schema,'table' as table_type from sys.exa_all_tables
+		select table_name,table_schema,'table' as table_type from sys.exa_user_tables
 		union
-		select view_name, view_schema,'view' from sys.exa_all_views
+		select view_name, view_schema,'view' from sys.exa_user_views
```

Comment:
this had been changed in a previous bugfix. what is the reason to change back to exa_user_views?

Reply:
Switched it back. There was no need to change it.
Exa_user shows only objects owned by user.
Exa_all shows all available to user, so it makes more sense in this case.